tests/test_planning.py

```diff
@@ -514,7 +514,7 @@ def test_policy_guided_sesame():
     pick_nsrt = nsrt_name_to_nsrt["Pick"]
     place_nsrt = nsrt_name_to_nsrt["Place"]
 
-    # When using a perfect policy, we should only expand nodes along the plan.
+    # When using a perfect policy, we should only expand the first node.
```

Comment:
i don't get it, why did this change? how can you not expand all the nodes to the goal?

Reply:
we create all the nodes to the goal, but we don't need to expand them